packages/expandable/index.js

```diff
@@ -65,7 +61,8 @@ class WarpExpandable extends kebabCaseAttributes(WarpElement) {
       ::slotted(:last-child) {
         margin-bottom: 0px !important;
       }
-    `,
+      @unocss-placeholder
```

Comment:
Based on your comment `Add an additional uno-setup for development-mode in the vite-config.js-file that includes mode: 'shadow-dom' and safeList component-classes, making it possible to test component-classes locally in elements-repo.`, since this is only needed for debug purpose, maybe we don't need to add it to the bundle?